lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
Do you mean if the `raygun.send(err)` call throws an error? Have you tried doing:

`raygun.send(err).then(...).catch(...)`